docs/website/docs/deployment-configurations/bare-metal.md

```diff
@@ -0,0 +1,108 @@
+# Run on a Bare-Metal Platform
+
+IREE supports CPU model execution on a bare-metal platform. That is, a platform
+without operating system support and the executable runs on the machine mode.
+The IREE compiler follows the path to generate the code runs on the
+[dynamic library CPU HAL driver](https://google.github.io/iree/deployment-configurations/cpu-dylib/)
+and a subset of runtime library can be used to execute the model.
+
+## Prerequisites
+
+Out-of-tree bare-metal platform tools and source code for the system should be
+ready, such as
+
+* Compilation toolchain
+* Platform linker script
+* Firmware libraries
+
+Please follow the
+[instructions](https://google.github.io/iree/deployment-configurations/cpu-dylib/#get-compiler-for-cpu-native-instructions)
```

Comment:
Does a relative path work here?

```suggestion
[instructions](./cpu-dylib.md#get-compiler-for-cpu-native-instructions)
```

Seems like it should: https://mkdocs.readthedocs.io/en/0.10/user-guide/writing-your-docs/#linking-documents

That would have the advantage of working when developing locally or publishing to a different domain (such as your own github pages).

Reply:
Done. Thanks for the tip!